content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -231,5 +210,5 @@ CNI plugin only performs actions to setup the application pod's traffic redirect
 sidecar (using `iptables` in the pod's network namespace).
 
 {{< warning >}}
-This _should_ have no effect on the operations performed by the base CNI plugin configuring the pod's networking setup, although not all CNI's have been validated.
+The Istio CNI plugin should have no effect on the operations performed by the base CNI plugin configuring the pod's networking setup, although not all CNI plugins have been validated.
```

Comment:
I tried to untangle the passive voice here as well. It's difficult because you have two plugins involved. What is the core message you want to tell readers? Is there anyway of phrasing this a bit more clearly?
```suggestion
The base CNI plugin that configures the pod's networking setup performs operations that the Istio CNI plugin should not affect, although not all CNI plugins have been validated.
```

Reply:
I'm suggesting this:
> The Istio CNI plugin should not interfere with the operations of the base CNI plugin that configures the pod's networking setup, ...